### ISSUE TITLE
Error using the tinder scraper

### ISSUE THREAD

**Issue by lefeverela** [gcwelborn/tinder-scraper#6]
First of all thanks for the work being done on this this is very useful.

I have followed all the steps on your tutorial:
1) Download the code
2) Use tindercred-master to get my facebook login and token
3) create an app on firebase
4) setup the config.js file like this (actual facebook ID and token have been replaced by XXX)

var config =  {
  FaceBook_ID : 2XXXXXXXXXXXXXXX0,
  FaceBook_Token : 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
  Firebase : 'testapp',
  IntervalInSeconds: 30,
}

module.exports = config

5) run npm install and npm start, and I get the following messages:

> node app.js
5/5/2016, 11:02:37 PM: attempting to authenticate into tinder....
5/5/2016, 11:02:38 PM: tinder client authorized...
5/5/2016, 11:02:38 PM: ERROR: Error: Forbidden

So I was wondering what I did wrong? or maybe tinder updated their security and this is not possible anymore?

Any help on this would be useful,

Thanks

**Comment by lefeverela** [comment]
in case asked my node version is v4.4.4
downloaded there:
https://nodejs.org/en/

I am on windows 7

**Comment by lefeverela** [comment]
Just in case, I am still having the issue, thanks for your help

**Comment by gcwelborn** [comment]
hey @lefeverela, i'll test tinder-scraper on windows 7 this weekend and see if I can recreate the issue. There could have been breaking changes made to tinderjs, or tinder may have changed some data - I doubt that it's not possible anymore. It's also possible that windows is causing the problems - one issue by @leesander1 may have been resolved by switching OSX or Linux, but unfortunately he didn't post a followup after closing the issue

**Comment by lefeverela** [comment]
very much appreciated. Thanks for the help

**Comment by leesander1** [comment]
I apologize for not following up but I retried on an ubuntu system and got
the same results.

**Comment by lefeverela** [comment]
I thought of something this morning which looks like to have solve my problem:
In order to test the core I created a new tinder account (new facebook).
But I actually never logged in from my phone before with that acccount.
This morning I logged in with my phone, then try the script again and now it works.
I believe the script was not working because maybe no GPS coordinates where registered or something,

Thanks for the help. 
I will come back if I still have issues, thanks all
